Makefile

```diff
@@ -30,14 +39,44 @@ e2e: deploy
 $(TARGETS): vendor/modules.txt
 	./scripts/$@
 
-build-subctl: pkg/subctl/operator/common/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go")
+clean:
+	rm -f $(BINARIES) $(CROSS_BINARIES) $(CROSS_TARBALLS)
+
+build: operator-image binaries
+
+build-cross: $(CROSS_TARBALLS)
+
+binaries: $(BINARIES)
+
+operator-image: vendor/modules.txt
+	operator-sdk build quay.io/submariner/submariner-operator:$(VERSION) --verbose
+
+bin/subctl: bin/subctl-$(VERSION)-$(GOOS)-$(GOARCH)$(GOEXE)
+	ln -sf $(<F) $@
+
+dist/subctl-%.tar.xz: bin/subctl-%
+	mkdir -p dist
+	tar -cJf $@ --transform "s/^bin/subctl-$(VERSION)/" $<
+
+# Versions may include hyphens so it's easier to use $(VERSION) than to extract them from the target
+bin/subctl-%: pkg/subctl/operator/common/embeddedyamls/yamls.go $(shell find pkg/subctl/ -name "*.go") vendor/modules.txt
+	mkdir -p bin
+	GOOS=$(word $(guile (- $(words $(subst -, ,$@)) 1)),$(subst -, ,$@)); \
+	GOARCH=$(word 1,$(subst ., ,$(lastword $(subst -, ,$@)))); \
+	GOEXE=$(suffix $@); \
```

Comment:
Turns out GOEXE is not settable, and is determined and reported by Go.
Hence, there's nothing to gain by setting it (funnily, it wasn't even used in the original `build-cross` script)

Reply:
Yes, I’ve removed that now.